models/src/main/thrift/content/v1.thrift

```diff
@@ -882,6 +884,18 @@ struct Tag {
     17: optional string twitterHandle
 }
 
+struct QuizAtom {
+  1: required string id
+  2: required list<string> labels // required, but may be empty
+  3: required string defaultHtml
+  4: required quiz.QuizAtom data       // the atom payload
+//  6: required ContentChangeDetails contentChangeDetails
+ }
+
+/* an embedded content item with an independant lifecycle */
+struct Atoms {
+   1: optional QuizAtom quiz
+}
```

Comment:
we should not define those in this file but include the content atom thrift model (adding a dependency to the published jar) so the only changes in the file should be:

```thrift
include "content-atom.thrift"

17: optional Atoms atoms
```